nearcore/tests/load_configs.rs

```diff
@@ -0,0 +1,19 @@
+use near_primitives::runtime::config::ActualRuntimeConfig;
+use near_primitives::runtime::config_store::RuntimeConfigStore;
+use nearcore::config::mainnet_genesis;
+
+/// Checks that getting configs from RuntimeConfigStore gives backward compatible results with the
+/// previous way, i.e. taking runtime config from genesis and modifying it.
+#[test]
+fn test_mainnet_backwards_compatibility() {
+    let genesis = mainnet_genesis();
+    let genesis_runtime_config = genesis.config.runtime_config;
+    let actual_runtime_config = ActualRuntimeConfig::new(genesis_runtime_config);
+
+    let store = RuntimeConfigStore::new(None);
+    for protocol_version in [29u32, 34u32, 42u32, 50u32].iter() {
+        let old_config = actual_runtime_config.for_protocol_version(protocol_version.clone());
+        let new_config = store.get_config(protocol_version.clone());
+        assert_eq!(old_config, new_config);
+    }
+}
```

Comment:
This adds a new test binary, which non-trivially increases the build time:

```
# make sure deps are fully compiled
λ cargo test -q --no-run --test economics

# compile the test in isolation
λ cargo test --no-run --test load_configs
   Compiling nearcore v0.0.0 (/home/matklad/projects/nearcore/nearcore)
    Finished test [unoptimized + debuginfo] target(s) in 1.81s
```

That is, just this one test adds 1.81 seconds of compile time. 

Better to put this as a unit-test into some existing crate:

https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html